src/endpoint.cpp

```diff
@@ -453,6 +456,7 @@ void Endpoint::_add_sys_comp_id(uint8_t sysid, uint8_t compid)
                  sniffer_sysid,
                  fd);
     }
+    log_info("%s Adding sysid: %d compid: %d", _name.c_str() ,sysid, compid);
```

Comment:
Is this log line really needed? You'll get the list of connected components on the endpoint for the next routed message anyways.
If you think the line is useful, please include the endpoint ID (fd) in the log line like: "Endpoint [%d]%s: Adding %d/%d to known components"